all.md

```diff
@@ -0,0 +1,19 @@
+---
+permalink: /docs/all/
+layout: docs
+title: All of the Latest Documentation
+category: ignore
+---
+
+# All of the Electron Documentation
+
+Below is the latest version of documentation. See all [available versions](/docs/).
+
+---
+
+{% for doc in site.docs %}
```

Comment:
Yeah, it's nice for looping through your collections. The tricky bit was that if you have subdirectories within your collection there isn't a nice way to distinguish between them. 

Because `/latest` is a dupe of `/v0.27.0` there wasn't an easy way to filter them so that we just get one set (we'd get two of every document if we tried to filter on version since they have the same front matter). But they do have different file paths, and so that's what we use to get just one set of the latest to show. :sparkles: